src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -115,8 +148,93 @@ const Body = BaseForm.extend({
   }
 });
 
+const Footer = AuthenticatorFooter.extend({
+  props: {
+    enabled: ['boolean', false]
+  },
+
+  events: {
+    'click .js-cant-verify': function(e) {
+      e.preventDefault();
+      if (!this.props.enabled) {
+        return;
+      }
+      this.toggleLinks(e);
+    },
+  },
+
+  initialize: function() {
+    let links = _.resultCtx(this, 'links', this);
+    const footerInfo = _.resultCtx(this, 'footerInfo', this);
+    const hasBackToSignInLink = _.resultCtx(this, 'hasBackToSignInLink', this);
+
+    if (!Array.isArray(links)) {
+      links = [];
+    } else {
+      links = links.filter(l => $.isPlainObject(l));
+    }
+
+    if (this.options.appState.shouldShowSignOutLinkInCurrentForm(
+      this.options.settings.get('features.hideSignOutLinkInMFA') ||
+      this.settings.get('features.mfaOnlyFlow')) && hasBackToSignInLink) {
+      links = links.concat(getBackToSignInLink(this.options.settings));
+    }
+
+    links.forEach(link => {
+      this.add(Link, {
+        options: link,
+      });
+      if (link.name === 'cant-verify') {
+        if (this.options.appState.get('app') && this.options.appState.get('app').name === 'Okta_Authenticator') {
+          this.add(CantVerifyInfoOVEnrollmentFlowTemplate);
+        } else {
+          this.add(CantVerifyInfoVerifyFlowTemplate);
+        }
+      }
+    });
+
+    if (footerInfo) {
+      this.add(View.extend({
+        className: 'footer-info',
+      }));
+
+      this.add(footerInfo, '.footer-info');
+    }
+
+    this.listenTo(this.props.enabled, 'change:enabled', function(model, enable) {
+      this.$('.link').toggleClass('o-form-disabled', !enable);
+    });
+  },
+
+  links: function() {
+    let links = AuthenticatorFooter.prototype.links.apply(this, arguments);
+    links.push({
+      'label': loc('oie.verify.webauthn.cant.verify', 'login'),
+      'name': 'cant-verify',
+      'href': '#',
+      'aria-expanded': false,
+      'aria-controls': 'help-description-container',
+      'class': 'link help js-help',
```

Comment:
in here what is missing is 
```
clickHandler: () => {
  // toggle the view that has the instructions
}
```

What I am missing I believe is where to add the view that has the instructions. So probably what would be better is to change `Link` so we can have a different type (maybe `toggle-link`) and that will add a view during initialize. Or something along those lines